src/AWS.Deploy.CLI/ServerMode/Services/IDeploymentSessionStateServer.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AWS.Deploy.CLI.ServerMode.Services
+{
+    public interface IDeploymentSessionStateServer
+    {
+
+        public SessionState Get(string id);
```

Comment:
we don't need to use "public" because this is an interface

Reply:
Done